classes/local/api.php

```diff
@@ -686,16 +686,18 @@ public function supports_api_level($level) {
      */
     public function connection_test_url() {
```

Comment:
This test really does not do what it promises! There is no check at all if the thing that is speaking HTTP is really Opencast. Is this check necessary and can't it be always replaced by `connection_test_credentials`?